Common/Gfx/Util/D2DBitmapLoader.cpp

```diff
@@ -49,27 +49,37 @@ HRESULT D2DBitmapLoader::LoadBitmapFromFile(const Canvas& canvas, D2DBitmap* bit
 	}
 	bitmap->SetFileTime(fileTime);
 
-	Microsoft::WRL::ComPtr<IWICBitmapDecoder> decoder;
-	Microsoft::WRL::ComPtr<IWICBitmapFrameDecode> decoderFrame;
+	// Microsoft::WRL::ComPtr<IWICBitmapDecoder> decoder;
+	// Microsoft::WRL::ComPtr<IWICBitmapFrameDecode> decoderFrame;
 	Microsoft::WRL::ComPtr<IWICBitmapSource> source;
 
-	HRESULT hr = Canvas::c_WICFactory->CreateDecoderFromFileHandle(
-		(ULONG_PTR)fileHandle,
-		nullptr,
-		WICDecodeMetadataCacheOnDemand,
-		decoder.GetAddressOf());
-	if (SUCCEEDED(hr))
-	{
-		hr = decoder->GetFrame(0U, decoderFrame.GetAddressOf());
-		if (SUCCEEDED(hr))
-		{
-			hr = ConvertToD2DFormat(decoderFrame.Get(), source);
-		}
-	}
-	if (FAILED(hr)) return cleanup(hr);
-
-	const int orientation = GetExifOrientation(decoderFrame.Get());
-	bitmap->SetOrientation(orientation);
+	// HRESULT hr = Canvas::c_WICFactory->CreateDecoderFromFileHandle(

```

Comment:
This commented stuff is here because it shouldn't be removed.
A new function `LoadBitmapFromPluginData` is needed here, that will be identical to `LoadBitmapFromFile`, except these commented lined.